hack/lib/util.sh

```diff
@@ -144,6 +144,9 @@ kube::util::host_os() {
     Darwin)
       host_os=darwin
       ;;
+    FreeBSD)
+      host_os=freebsd
+      ;;
```

Comment:
You need to change `Unsupported host OS.  Must be Linux or Mac OS X.` as well, surely?